utils/progressbar/taskswithheadlineprog.go

```diff
@@ -10,6 +10,12 @@ type TasksWithHeadlineProg struct {
 	tasksProgressBar *TasksProgressBar
 	emptyLine        *mpb.Bar
 }
+type TasksWithTwoHeadLinesProg struct {
```

Comment:
1. Missing new line before the declaration of this struct.
2. This struct doesn't include a description like the the struct above it. Maintaining consistency in the code is very important.

Reply:
This struct is not needed so I deleted it.